exercises/transpose/example.js

```diff
@@ -0,0 +1,12 @@
+export default function transpose(text) {
+  const result = [];
+  text.forEach((line, lineNo) => {
+    line.split('').forEach((value, key) => {
+      if (typeof result[key] === 'undefined') {
+        result[key] = ' '.repeat(lineNo);
+      }
+      result[key] += value;
+    });
+  });
+  return result;
+}
```

Comment:
This all looks good, but it did make me wonder if it's possible to do this entirely functionally-- IE, without having an initial `result` array that gets mutated.  Not critical, but just thinking out loud...